packages/protocol/docs/tokenomics_objective_metrics.md

```diff
@@ -10,18 +10,18 @@ Based on the provided insights and inputs from Brecht and Hugo (zkpool), the fol
 
 1. **Taiko Token Centric**: The tokenomics design should primarily revolve around our Taiko token, rather than other tokens such as USD stable coins. An equilibrium should be achieved between proposer fees and prover rewards, aiming to maintain or even slightly decrease the supply of Taiko tokens over time.
 
-1. **Efficient Use of Proving Resources**: Provers should not squander computational resources on non-profitable tasks. If resources are wasted, provers will understandably require higher returns from accepted proofs to offset their losses. This would invariably raise the average cost of proofs.
+2. **Efficient Use of Proving Resources**: Provers should not squander computational resources on non-profitable tasks. If resources are wasted, provers will understandably require higher returns from accepted proofs to offset their losses. This would invariably raise the average cost of proofs.
```

Comment:
I prefer the original approach as markdown renderer will figure out the actual numbers automatically.